src/Models/Booking.php

```diff
@@ -190,19 +194,39 @@ public function bookingCategory()
         return $this->belongsTo(app('booking_category'));
     }
 
+    public function bookingSlot()
+    {
+        return $this->belongsTo(app('booking_slot'));
+    }
+
     public function subject(): Relation
     {
         return $this->morphTo();
     }
 
+    public function location()
+    {
+        return $this->belongsTo(app('location'));
+    }
+
+    public function rate()
+    {
+        return $this->belongsTo(app('rate'));
+    }
+
+    public function escaperoomRate()
```

Comment:
I don't think we want escaperoomRate here. I think it should just be using the Rate relation